app/controllers/index.rb

```diff
@@ -0,0 +1,31 @@
+get '/' do
+  @photos = Photo.all
+  erb :index
+end
+
+post '/' do
+  @user = User.create(first_name: params[:first_name], last_name: params[:last_name], username: params[:username], email: params[:email], password_hash: params[:password])
```

Comment:
you're using `create` and `save` here. You should be using `User.new(…)` and then `if @user.save` just as you are.